content/en/docs/instrumentation/js/getting-started/browser.md

```diff
@@ -40,26 +40,68 @@ Copy the following file into an empty directory and call it `index.html`.
 
 To create traces in the browser, you will need `@opentelemetry/sdk-trace-web`, and the instrumentation `@opentelemetry/instrumentation-document-load`:
 
-```shell
+{{< tabs TypeScript JavaScript >}}
+
+{{< tab >}}
+tsc --init
+npm install --save @opentelemetry/api @opentelemetry/sdk-trace-web @opentelemetry/instrumentation-document-load @opentelemetry/context-zone
+{{< /tab >}}
+
+{{< tab >}}
 npm init -y
 npm install --save @opentelemetry/api @opentelemetry/sdk-trace-web @opentelemetry/instrumentation-document-load @opentelemetry/context-zone
-```
+{{< /tab >}}
+
+{{< /tabs >}}
 
 ### Initialization and Configuration
 
-Create a empty file called `document-load.js` and add the following code to your html right before the body end tag:
+Create a empty file called `document-load.ts` or `document-load.js` (Based on if you are using typescript or javascript) and add the following code to your html right before the body end tag:
 
-```html
+{{< tabs TypeScript JavaScript >}}
+
+{{< tab >}}
+<script type="module" src="document-load.ts"></script>
+{{< /tab >}}
+
+{{< tab >}}
 <script type="module" src="document-load.js"></script>
-```
+{{< /tab >}}
+
+{{< /tabs >}}
 
 We will add some code that will trace the document load timings and output those as OpenTelemetry Spans.
 
 ### Creating a Tracer Provider
 
-Add the following code to the `document-load.js` to create a tracer provider, which brings the instrumentation to trace document load:
+Add the following code to the `document-load.ts` or `document-load.js` to create a tracer provider, which brings the instrumentation to trace document load:
 
-```javascript
+{{< tabs TypeScript JavaScript >}}
+
+{{< tab >}}
+/* document-load.ts */
+import { WebTracerProvider } from '@opentelemetry/sdk-trace-web';
+import { DocumentLoadInstrumentation } from '@opentelemetry/instrumentation-document-load';
+import { ZoneContextManager } from '@opentelemetry/context-zone';
+import { registerInstrumentations } from '@opentelemetry/instrumentation';
+
+const provider:WebTracerProvider = new WebTracerProvider();
```

Comment:
Is this type declaration really necessary? Can't TS infer the type automatically? (I haven't worked with TS in a while so I don't know if it can do something like that now -- also, I might be confusing it with [Dart](https://dart.dev/).)

If the type declaration isn't necessary, then the two code blocks are identical, and so we should avoid tabs, just present the code that is valid for both. My 2 cents.

Reply:
@chalin you are right I will remove the tabbing and make it as a single code then